pkg/loftconfig/client.go

```diff
@@ -19,7 +19,7 @@ import (
 
 var backoff = wait.Backoff{
 	Steps:    4,
-	Duration: 300 * time.Millisecond,
+	Duration: 400 * time.Millisecond,
```

Comment:
do we need an env var to specify this? Why did we increase? Does this depend on how far the loft platform is from the client?

Reply:
Just to be sure :) 
Don't think this needs to be user controllable as it's just a brief delay in starting the credentials server in the workspace